src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -26,20 +33,25 @@
 // Section
 .pf-l-toolbar__section {
   flex-basis: 100%;
-
-  // Add margin-top for stacked sections
-  &:not(:first-child) {
-    margin-top: var(--pf-l-toolbar__section--MarginTop);
-  }
 }
 
 // Group
-.pf-l-toolbar__group:not(:last-child) {
-  margin-right: var(--pf-l-toolbar__group--MarginRight);
+.pf-l-toolbar__group {
+  padding: var(--pf-global--spacer--xl);
```

Comment:
This padding is being overwritten by the [workspace CSS](https://github.com/patternfly/patternfly-next/blob/master/src/site/workspace.scss#L227), but I don't think a padding is necessary.

Also, you shouldn't assign a global variable directly to a CSS property. You should always assign the global variable to a new variable specific for this layout, then use the layout specific variable as the value for the css property. You created that variable (`--pf-l-toolbar__group--Padding`), and need to use that here instead of `--pf-global--spacer--xl`

Reply:
Padding settings as been removed from the latest build.